### PR TITLE
qa/tasks: Changing default mClock profile to high_recovery_ops

### PR DESCRIPTION
**Pull request by amathuria**
Change the mClock profile in tests to high_recovery_ops in order to finish scrubs in time

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by amathuria** [comment]
@idryomov  I have made this change to fix the issue RBD tests have been facing with very long scrub times in teuthology tests.

Waiting for - http://pulpito.front.sepia.ceph.com/amathuri-2023-05-11_04:31:10-rbd-main-distro-default-smithi/ to pass

**Review by idryomov — COMMENTED** [review]
This is something we want for _all_ teuthology jobs, right?  If so, I would rather set it in `qa/tasks/cephadm.conf` and `qa/tasks/ceph.conf.template` to avoid littering cluster definition snippets with this option.

**Comment by amathuria** [comment]
> This is something we want for _all_ teuthology jobs, right? If so, I would rather set it in `qa/tasks/cephadm.conf` and `qa/tasks/ceph.conf.template` to avoid littering cluster definition snippets with this option.

Thanks! Does this look okay?

**Review by neha-ojha — APPROVED** [review]

File: qa/tasks/ceph.conf.template
Comment (idryomov) [inline qa/tasks/ceph.conf.template:72]: Nit: since the lack of this setting causes "Exiting scrub checking -- not all pgs scrubbed" errors, it would make sense to place it near scrub and recovery options, e.g. after `osd scrub max interval = 600`.  It's certainly not a debug option...

File: qa/tasks/cephadm.conf
Comment (idryomov) [inline qa/tasks/cephadm.conf:51]: ditto

**Review by idryomov — COMMENTED** [review]
Please mention "Exiting scrub checking -- not all pgs scrubbed" error in the commit message so that it's clear what the symptom was.

**Comment by idryomov** [comment]
Confirming that this PR fixes the issue from the RBD suite side.

**Comment by ljflores** [comment]
Rados suite summary available here: https://tracker.ceph.com/projects/rados/wiki/MAIN#httpstrellocomc1EFSeXDn1752-wip-yuri10-testing-2023-05-16-1243

**Comment by ljflores** [comment]
jenkins test api

**Comment by ljflores** [comment]
@yuriw might want to hold off on merging this since @idryomov is checking something from the RBD side.

**Comment by yuriw** [comment]
@amathuria @idryomov 
this was tested and can be merged if rados was sufficient, otherwise kick it back to the queue.

ref: https://trello.com/c/1EFSeXDn

**Comment by idryomov** [comment]
> @idryomov is checking something from the RBD side

It's OK from the RBD side but there are outstanding nits (non-blocking but it would be nice if they were addressed).

**Comment by idryomov** [comment]
This fixes an issue introduced in https://github.com/ceph/ceph/pull/49975.

**Comment by idryomov** [comment]
> there are outstanding nits (non-blocking but it would be nice if they were addressed)

Also a tracked ticket is needed for backporting to quincy and reef (that's where commits from https://github.com/ceph/ceph/pull/49975 are present as of today).

**Comment by amathuria** [comment]
> > @idryomov is checking something from the RBD side
> 
> It's OK from the RBD side but there are outstanding nits (non-blocking but it would be nice if they were addressed).

@idryomov Sure I'll fix them

**Comment by amathuria** [comment]
> > there are outstanding nits (non-blocking but it would be nice if they were addressed)
> 
> Also a tracker ticket is needed for backporting to quincy and reef (that's where commits from #49975 are present as of today).

@idryomov I have added the tracker ticket to the commit message as well

**Review by idryomov — APPROVED** [review]

**Pull request merged**
